conventions/documentation/alerting.md

```diff
@@ -0,0 +1,72 @@
+---
+parent: Conventions
+---
+
+# Alerting
+
+## Inline Alerts
+Inline Alerts can be used throughout the console to notify users in context that something is happening, happened, or needs their attention.
+
+### Inline Alerts in Details Views
+
+#### Above navigation tabs
+Inline alerts can appear above the navigation tabs when there is an important piece of information that applies to the entire resource, and the user should be made aware of regardless what tab they are looking at.
+
+![inline alert details above tabs](../images/alerting-inline-details-above.png)
```

Comment:
I wonder if we should include an alternative design here too for multiple errors being surfaced in different tabs. In CNV we are listing out the different areas that have pending changes as a bulleted list of links to allow the user to easily navigate to those areas. WDYT?

Reply:
That sounds like it could be worth including, mind sending me that design so I can add it?

Reply:
Got it in there, thanks 👍